public/app/features/logs/components/log-context/LogRowContextModal.test.tsx

```diff
@@ -190,11 +218,14 @@ describe('LogRowContextModal', () => {
       <LogRowContextModal row={row} open={true} onClose={() => {}} getRowContext={getRowContext} timeZone={timeZone} />
     );
 
-    expect(
-      screen.queryByRole('button', {
-        name: /open in split view/i,
-      })
-    ).not.toBeInTheDocument();
+    // await waitFor(() => {expect(getRowContext).toHaveBeenCalled();});
```

Comment:
do you want to keep this commented out line?

Reply:
removed in d6f7ada20c331a7894487a87c919eeb0423b6f46